tests/run.sh

```diff
@@ -169,6 +169,9 @@ start_awesome() {
     awesome_pid=$!
     cd - >/dev/null
 
+    # Do not put too much load on this, it will deadlock
+    sleep 0.5
```

Comment:
I plan to remove this commit from the final version and do a PR. For now, I still need it, otherwise I can't run the tests and have to wait 7 minutes for Travis.